<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPasswdWindow</class>
 <widget class="QWidget" name="ModifyPasswdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModifyPasswdWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>100</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>60</y>
     <width>42</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>旧密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oldpasswd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QToolButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QToolButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newpasswd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_repeatpasswd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_oldpasswd</tabstop>
  <tabstop>lineEdit_newpasswd</tabstop>
  <tabstop>lineEdit_repeatpasswd</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
